Islandora-CLAW/islandora_demo
# Using iana:describedby for altering media jsonld

**GitHub Issue**: Part of https://github.com/Islandora-CLAW/CLAW/issues/662

Required by https://github.com/Islandora-CLAW/islandora/pull/136

# What does this Pull Request do?

Configures `MappingUriPredicateReaction` to use `iana:describedby` instead of `schema:sameAs`.

# How should this be tested?
See https://github.com/Islandora-CLAW/islandora/pull/136 for testing instructions.

# Interested parties
@Islandora-CLAW/committers
